client/src/components/Event.test.tsx

```diff
@@ -0,0 +1,42 @@
+
+import { render, screen } from "@testing-library/react";
+import { Event, DateRecord } from "./Event";
+
+describe("Event component", () => {
```

Comment:
Vsechny describe, expect a it nejsou spravne detekovany TS, hlasi napr.
```
TS2304: Cannot find name expect
```

je treba nainstalovat typy:
```
 yarn add -D @types/jest
```

pak TS bude rozumet.